### PR TITLE
KFSPTS-11617: Changes to handling of PMW statuses and dupes

### PR DESCRIPTION
**Pull request by cah292**
There are two PRs: A code one and a SQL one. Both need to go together.

This adds the PMW status transition changes that came out of the PMW meeting earlier this week. Please let me know if you find any problems with the new approach.

There may be a few spots that I need to revise more. The reporting of duplicate requests will label them as needing manual entry (thus printing the verbose content in the report); will this add unnecessary clutter to the report? In addition, PaymentWorks doesn't appear to allow requests in "Approved" status to be rejected or returned to vendor; do we not need the timestamp-based duplicate checks as a result, or do we need further re-evaluation of how we're handling PMW status transitions?

The timestamp-based duplicate checks are currently based on when the request was last submitted. Would it make more sense to do it based on when the request last had a status transition?

I also wasn't sure if I was handling the naming conventions properly for the changes to the existing status-updating methods. Please let me know if you have suggestions on what names to use instead.

### CODE REVIEW

**Comment by nkimble** [comment]
retest this please

**Comment by nkimble** [comment]
retest this please

**Comment by nkimble** [comment]
retest this please

File: src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksPropertiesConstants.java [hunk 1]
```diff
@@ -10,6 +10,7 @@
         public static final String KFS_ACH_PROCESSING_STATUS = "kfsAchProcessingStatus";
         public static final String PMW_TRANSACTION_TYPE = "pmwTransactionType";
         public static final String SUPPLIER_UPLOAD_STATUS = "supplierUploadStatus";
+        public static final String LAST_SUBMITTED_TIMESTAMP = "lastSubmittedTimestamp";
```
[the remark this answers is not in the document]
Comment (cah292) [reply src/main/java/edu/cornell/kfs/pmw/batch/PaymentWorksPropertiesConstants.java]: Thanks for pointing that out. I'll get the property name updated accordingly.

**Comment by cah292** [comment]
After further discussion with PMW and our functionals, we don't need to implement special reprocessing for PMW-Approved vendors. The functionals will manually enter data as needed in the rare event that the KFS-side processing fails.

Given that, we don't need to implement the the new resubmission handling because any resubmits should take place in the pre-KFS-read phase of the PMW routing. We can continue to use "Approved" and "Processed" status as before, but we'll still need to remove the "Rejected" status updates from our processing.

I'm closing this PR due to how the type and scope of the changes have changed drastically. I'll create a new PR for a new user story when ready.

**Pull request closed without merging**